src/components/ProfileEditImage.js

```diff
@@ -0,0 +1,23 @@
+export default class ProfileEditImage {
```

Comment:
Needs correction: you don't need such class, use PopupWithForm for the avatar popup